collect_app/src/main/java/org/odk/collect/android/activities/GeoShapeOsmMapActivity.java

```diff
@@ -203,7 +203,7 @@ protected void onResume() {
 
     @Override
     public void onBackPressed() {
-        if (mapMarkers != null && mapMarkers.size() > 0) {
+        if (!mapMarkers.isEmpty()) {
```

Comment:
This will fail if mapMarkers is null. Is the null check in the original unneeded?

Reply:
`mapMarkers` is defined as final and initialized with empty `ArrayList` so it can't be null

Reply:
Excellent!